QuasarHelios2015/src/org/team1540/quasarhelios/AutoEjector.java

```diff
@@ -26,22 +30,19 @@ public static BooleanStatus create() {
 
     @Override
     protected void autonomousMain() throws AutonomousModeOverException, InterruptedException {
-        try {
-            done.set(false);
-            Elevator.setMiddle.event();
+        Elevator.setBottom.event();
 
-            waitUntil(Elevator.bottomLimitSwitch);
+        waitUntil(Elevator.atBottom);
 
-            Rollers.open.set(true);
-            Rollers.direction.set(false);
-            Rollers.running.set(true);
+        Rollers.open.set(true);
+        Rollers.direction.set(false);
+        Rollers.running.set(true);
 
-            waitUntil(BooleanMixing.invert(AutoLoader.crateInPosition));
-            waitForTime(timeout);
+        waitUntil(BooleanMixing.invert(AutoLoader.crateInPosition));
+        waitForTime(timeout);
 
-            Rollers.running.set(false);
-        } finally {
-            done.set(true);
-        }
+        Rollers.running.set(false);
+        
+        running.set(false);
```

Comment:
Won't this cause an issue if this doesn't get called - for example, if some error occurs? Then we'll never be able to start the AutoEjector again...